scripts.d/.plan-cfd-ng-dev

```diff
@@ -5,7 +5,7 @@
 s-salome-ng/neptune.cfd
 
 s-libccmio:92f08637
-s-coolprop:#6.1.0
+s-coolprop:#6.4.3
```

Comment:
This is asking to use version 6.4.3 from an archive. Do we have such version on our MinIO?

Reply:
Responding to myself, yes we have and I think I've uploaded it some time ago :)